server.js

```diff
@@ -11,11 +11,12 @@ const listener = app.listen(port, () => {
 // serve all static files
 app.use(express.static('.', {
     setHeaders: (res, path) => {
-        res.set("Access-Control-Allow-Origin", "*");
+        res.set('Access-Control-Allow-Origin', '*');
```

Comment:
this allows 3p websites to access content of requests (we use that for request blocking)